src/test/java/org/generationcp/middleware/service/impl/inventory/LotIDGeneratorTest.java

```diff
@@ -0,0 +1,68 @@
+package org.generationcp.middleware.service.impl.inventory;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.generationcp.middleware.manager.InventoryDataManagerImpl;
+import org.generationcp.middleware.pojos.ims.Lot;
+import org.generationcp.middleware.pojos.workbench.CropType;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Arrays;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class LotIDGeneratorTest {
```

Comment:
@vmaletta these tests shold be located at InventoryDataManagerImplTest, please rename this class if it does not exist

Reply:
Renamed!